frontend/src/common/dto/channel.dto.ts

```diff
@@ -0,0 +1,21 @@
+export class ChannelDto {
```

Comment:
So are these still relevant after @fyusuf-a 's refactor of Dto ?

Reply:
Yeah, I was having trouble getting the backend Dtos to work here. I'll try to look at that with Florian, to see if I was doing something wrong

Reply:
Opened #163